Reference/steering-display/readme.md

```diff
@@ -0,0 +1,59 @@
+# EVE_Test_Arduino_PlatformIO Example
```

Comment:
Modify readme to explain how this reference code works- what is different from the template?

Reply:
TODO

Reply:
This was just a working copy. I have since deleted it from the branch. Not meant to be any different from what is in templates folder.